sbi/inference/base.py

```diff
@@ -101,7 +100,7 @@ def __init__(
                 parameters, e.g. which ranges are meaningful for them. Any
                 object with `.log_prob()`and `.sample()` (for example, a PyTorch
                 distribution) can be used.
-            device: torch device on which to compute, e.g. gpu or cpu.
+            device: torch device on which to train the neural net, e.g. gpu or cpu.
```

Comment:
Since the device will be passed to the posterior, it will also be where e.g. sampling is done, no? If yes, we should mention that all `posterior` operations will also be performed on `device`.